README.md

```diff
@@ -91,6 +94,12 @@ You can use this option to pass a custom location to `versionist.conf.js`.
 You can use this option to perform a dry run to see what changes _would_ be
 made by versionist, without actually changing any files.
 
+### `get <target>`
+
+You can use this option to return the latest documented version (according to `getChangelogDocumentedVersions`) or a git reference which is produced by applying `getGitReferenceFromVersion` to the latest documented version.
+This command will not run versionist.
```

Comment:
LOL, "This command will not run versionist" sounds a little funny :laughing: Did you mean something like "This option will not cause any changes" ?

Reply:
Hehehe that is true, I will update it :)